src/log/ModUdp.cc

```diff
@@ -43,15 +43,6 @@ logfile_mod_udp_write(Logfile * lf, const char *buf, size_t len)
     ssize_t s;
     s = write(ll->fd, (char const *) buf, len);
     fd_bytes(ll->fd, s, FD_WRITE);
-#if 0
-    if (s < 0) {
-        int xerrno = errno;
-        debugs(1, DBG_IMPORTANT, "logfile (udp): got errno (" << errno << "):" << xstrerr(xerrno));
-    }
```

Comment:
The error display part should remain for admin troubleshooting. Though it can be reduced to debug level 3 and have one of the fancy FadingCounter to reduce log verbosity.
Please at least add a TODO above that.

Reply:
Since Amos thinks this code is valuable, I recommend keeping it "as is" (i.e. removing nothing and changing nothing) in _this_ PR. Properly enabling this code will require some work, and we should not do that in _this_ PR. Adding a TODO comment is fine, of course.